README.md

```diff
@@ -231,4 +231,4 @@ If you have questions or comments, please create an issue.
 
 ## License
 
-<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/by-sa/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
+<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/by-sa/4.0/88x31.png" /></a><br />This work is licensed under the <a rel="license" href="https://github.com/freeCodeCamp/how-to-contribute-to-open-source/blob/main/LICENSE.md">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
```

Comment:
```suggestion
<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/by-sa/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="https://github.com/freeCodeCamp/how-to-contribute-to-open-source/blob/main/LICENSE.md">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
```
The correct term is "This work is licensed under a Creative Commons
Attribution-ShareAlike 4.0 International License". It is already correct.